spec/spec_helper.rb

```diff
@@ -33,12 +38,24 @@ def initialize(id = 'content_type')
 end
 
 class EntryDouble < Contentful::DynamicEntry
-  attr_reader :id, :content_type, :fields
+  attr_reader :id, :content_type
 
-  def initialize(id = '', content_type = ContentTypeDouble.new, fields = {})
+  def initialize(id = '', content_type = ContentTypeDouble.new, fields = {}, locales = false)
     @id = id
     @content_type = content_type
-    @fields = fields
+    @fields = {}
```

Comment:
No, because the @fields needs to exist previously for the case in which `locales = false`, which is actually all but 1 test.